test/e2e-tests.sh

```diff
@@ -25,10 +28,8 @@ fi
 save_release_artifacts || fail_test "Failed to save release artifacts"
 
 if ! ${LOCAL_DEVELOPMENT}; then
-  scale_controlplane kafka-controller kafka-webhook-eventing eventing-webhook eventing-controller
-  wait_until_pods_running knative-eventing || fail_test "Pods in knative-eventing didn't come up"
-  apply_chaos || fail_test "Failed to apply chaos"
-  apply_sacura || fail_test "Failed to apply Sacura"
+  "${ROOT_DIR}"/hack/run.sh deploy-sacura || fail_test "Failed to apply Sacura"
+  "${ROOT_DIR}"/hack/run.sh deploy-chaos || fail_test "Failed to apply chaos"
```

Comment:
You still have these depending on run.sh

Reply:
Done